### DISCUSSION TITLE
Static Resource PrimaryXXX not found

### DISCUSSION THREAD

**Discussion by DrCQ** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2840] [Q&A]
I'm migrating my Framework 4.8 project with MaterialDesingThemes/Colors V3.1.0/1.2.3 to NET6.0 with MaterialDesignThemes/Colors V4.5.0/2.06 and run in trouble with my SolidColorBrush definitions. The system runs in an Exception "Resource not fond" here:

`<SolidColorBrush x:Key="ForegroundSubItem" Color="{StaticResource Primary100}"/>`

What is wrong? I didn't find anything in Wiki. Any idea how to access the PrimaryXXX and AccentXXX colors?

**Comment by Keboo** [comment]
Can you share your App.xaml, specifically on how you are setting the theme colors? There were quite a few theming/brush changes in the 4.x switch. Specifically, if you want to have resources like the `Primary*` colors defined you need to specify the color theme referencing the appropriate color dictionaries. You can see an example of that using [the demo app](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/6d13bad7ccb764217207a455c63b31b6b480dab1/MainDemo.Wpf/App.xaml#L21-L25). All of those color resources are still available, but the helper dictionaries (`BundledTheme` and `CustomColorTheme`) do not automatically add them any more, they must be explicitly referenced.
